assets/blocks/course-categories-block/block.json

```diff
@@ -0,0 +1,56 @@
+{
+	"$schema": "https://schemas.wp.org/trunk/block.json",
+	"apiVersion": 2,
+	"name": "sensei-lms/course-categories",
+	"category": "sensei-lms",
+	"textdomain": "sensei-lms",
+	"attributes": {
+		"align": {
+			"type": "string",
+			"default": "left"
+		},
+		"categoryStyle": {
+			"type": "object"
+		},
+		"fontSize": {
+			"type": "string",
+			"default": "small"
+		},
+		"style": {
```

Comment:
Just curious what this attribute is for? I couldn't trace where it was being used.

Reply:
It is used to define margins on our block, by default, there is no margin configured.
I could define it directly on the CSS, but in this way, the users can customize it.

Reply:
> I could define it directly on the CSS, but in this way, the users can customize it.

How does the user customize it? I'm not seeing anything for margin in the block settings. Attributes are normally used for storing block settings. I have a similar question about `categoryStyle` and the necessity of it as well.

I'm focusing on the attributes here because it's important to try to get them right the first time. They're not something that can easily be changed later since you have to deprecate the block first, and then update attributes.